app/scss/adminPage.scss

```diff
@@ -309,4 +309,8 @@ input[type="number"] {
   &:hover {
     color: #808080;
   }
+}
+
+.white-background {
```

Comment:
rename class to something more semantic that says this is the header of the expanded accordion

Reply:
also done!